### DISCUSSION TITLE
Regarding Turn by turn navigation

### DISCUSSION THREAD

**Discussion by groomkrunal** [maplibre/maplibre#172] [General]
Hello,

I implemented https://github.com/maplibre/maplibre-navigation-android example from here. But it will show only mock route navigation i want actual navigation. Can anyone please help me with that ?

**Comment by HarelM** [comment]
You'll need some backend for that, graphhopper, valhalla, osrm etc.
You can also check maplibre-direction package.

**Reply by groomkrunal** [reply]
Thank you for reply. We have integrated that but when we integrate maplibre android library it only show mock navigation (turn by turn) but I want real turn by turn navigation.
